docs/msgextras.md

```diff
@@ -0,0 +1,66 @@
+---
+id: msgextras
+title: Message Extras
+---
+
+Extras in messages can be used to extend clients. This page lists properties supported by the official gotify/\* clients.
+
+Extras are stored in a key-value scheme and are only accepted in `POST /message` requests
+with `application/json` content-type. Keys should be in the following format: `<top-namespace>::[<sub-namespace>::]<action>`
+
+Here's an example how the message with extras looks like. More information can be found in the [swagger-docs](../api-docs).
+
+```json
+{
+  "title": "my title",
+  "message": "my message",
+  "extras": {
+    "home::appliances::thermostat::change_temperature": { "temperature": 23 },
+    "home::appliances::lighting::on": { "brightness": 15 }
+  }
+}
+```
+
+## Message ContentType
```

Comment:
```suggestion
## Message Rendering
```

In the future, there might be more than `contentType` under `client::display` namespace that specifies other perspectives of the display of messages.